packages/react-conformance/src/isConformant.ts

```diff
@@ -13,7 +13,11 @@ export function isConformant(testInfo: IsConformantOptions) {
     throw new Error(`Path ${componentPath} does not exist`);
   }
 
-  const parser = withCustomConfig(tsconfigPath, {});
+  // Props need to be filtered since react-docgen shows all the props including props
+  // that aren't specified in the component.
```

Comment:
I'd modify this slightly--the problem isn't all inherited props (there are actually some cases where we want inherited props), just native props (ones from DOM elements) or certain React built-ins like ref.
```suggestion
  // inherited native props or React built-in props.
```